website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -0,0 +1,281 @@
+ 
+## Migrating your CR's
+
+Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`. In our example 
+the `config/samples/cache_v1alpha1_memcached.yaml` will look like:
+
+```yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-sample
+spec:
+  # Add fields here
+  size: 3
+```
+
+## Migrating your Watches
+
+Update the `watches.yaml` file with your `roles/playbooks` as its customizations from your previous project. In our example it will look like:
+
+Replace:
+```yaml
+---
+# Use the 'create api' subcommand to add watches to this file.
+- version: v1alpha1
+  group: cache.example.com
+  kind: Memcached
+  # FIXME: Specify the role or playbook for this resource.
+# +kubebuilder:scaffold:watch
+```
+With:
+
+```yaml
+---
+# Use the 'create api' subcommand to add watches to this file.
+- version: v1alpha1
+  group: cache.example.com
+  kind: Memcached
+  role: memcached
+# +kubebuilder:scaffold:watch
+```
+
+**NOTE**: Do not remobe the [Maker][marker] `+kubebuilder:scaffold:watch`it allows the tool upddate the watches file when new API's are created. 
+
+## Migrating your Molecule
+
+If you are using [Molecule][molecule] in your project then, ensure you will need t migrate your code to the new project to the `molecule/` directory and then, beware: 
+
+By default, the Molecule directory was:
+```shell  
+.
+├── cluster
+│   ├── converge.yml
+│   ├── create.yml
+│   ├── destroy.yml
+│   ├── molecule.yml
+│   ├── prepare.yml
+│   └── verify.yml
+├── default
+│   ├── converge.yml
+│   ├── molecule.yml
+│   ├── prepare.yml
+│   └── verify.yml
+├── templates
+│   └── operator.yaml.j2
+└── test-local
+    ├── converge.yml
+    ├── molecule.yml
+    ├── prepare.yml
+    └── verify.yml
+```
+
+Now, it will be like: 
+```
+ $ tree
+.
+├── default
+│   ├── converge.yml
+│   ├── create.yml
+│   ├── destroy.yml
+│   ├── kustomize.yml
+│   ├── molecule.yml
+│   ├── prepare.yml
+│   ├── tasks
+│   │   └── memcached_test.yml
+│   └── verify.yml
+└── kind
+    ├── converge.yml
+    ├── create.yml
+    ├── destroy.yml
+    └── molecule.yml
+```
+
+- Replace the var `${MOLECULE_EPHEMERAL_DIRECTORY}` with `${MOLECULE_PROJECT_DIRECTORY}`
+- Use now `${KUBECONFIG:-"~/.kube/config"}` to get the kubeconfig instead of `${MOLECULE_EPHEMERAL_DIRECTORY}/kubeconfig`
+- Ensure that the `provisioner.host_vars.localhost` has the following `host_vars`: 
+
+```
+....
+    host_vars:
+      localhost:
+        ansible_python_interpreter: '{{ ansible_playbook_python }}'
+        config_dir: ${MOLECULE_PROJECT_DIRECTORY}/config
+        samples_dir: ${MOLECULE_PROJECT_DIRECTORY}/config/samples
+        operator_image: ${OPERATOR_IMAGE:-""}
+        operator_pull_policy: ${OPERATOR_PULL_POLICY:-"Always"}
+        kustomize: ${KUSTOMIZE_PATH:-kustomize}
+...
+```
+
```

Comment:
Hi @fabianvf @asmacdo

Feel free to check the doc. I might need a help with the molecule section because it changed a lot.
However, I am trying to figure out it. See the sample migrated too in the pr: https://github.com/operator-framework/operator-sdk-samples/pull/138/files